src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -132,11 +132,17 @@ function getListBlockType(
 function getBlockMapSupportedTags(
   blockRenderMap: DraftBlockRenderMap
 ): Array<string> {
+  // Some blocks must be treated as unstyled when not present on the blockRenderMap
   const unstyledElement = blockRenderMap.get('unstyled').element;
-  return blockRenderMap
+  const defaultUnstyledSet = new Immutable.Set(['p']);
```

Comment:
Yeah, agreed I think that's the best option. That makes it one step closer to a schema, could also supply a function. In some cases custom elements will only match a particular type of html elements. Maybe a user will have a different block type for certain divs with different classes. 

I can update this PR to cover this.  What do you reckon ?